client/src/nls/es/locale.js

```diff
@@ -186,7 +211,7 @@ define({
 
     "Edit history tags": "Editar etiquetas de historial",
 
-    "Edit history Annotation": "Editar anotación del historial",
+    "Edit history annotation": "Editar historial de anotación",
```

Comment:
If what we are doing here is editing the annotation of a history, I would say `Editar anotación del historial` otherwise, `Editar historial de anotación` would mean we are going to edit the history of an annotation.

```suggestion
    "Edit history annotation": "Editar anotación del historial",
```

Reply:
That sounded odd to me too from my limited grammar, but I was just copying and pasting the team's translations. I can definitely see how that could get mistaken/mistranslated given the limited english text.